src/Contract/Monad.purs

```diff
@@ -77,40 +79,47 @@ import Wallet (mkNamiWalletAff)
 -- | All useful functions written in `QueryM` should be lifted into the
 -- | `Contract` monad and available in the same namespace. If anything is
 -- | missing, please contact us.
-newtype Contract (a :: Type) = Contract (QueryM a)
+newtype Contract' (r :: #Type) (a :: Type) = Contract (QueryMExtended r a)
```

Comment:
Isn't `# Type` notation going to be deprecated? If so, we should use `Row Type` instead.